compat/src/forwardRef.js

```diff
@@ -1,16 +1,5 @@
-import { options } from 'preact';
 import { assign } from './util';
 
-let oldVNodeHook = options.vnode;
-options.vnode = vnode => {
-	if (vnode.type && vnode.type._forwarded && vnode.ref) {
-		vnode.props.ref = vnode.ref;
-		vnode.ref = null;
-	}
-
-	if (oldVNodeHook) oldVNodeHook(vnode);
-};
```

Comment:
Moving this code from compat to core shaves 47 bytes out of compat and only adds 6 bytes to core so I thought it was worth it.

Reply:
And if we didn't install react compat in Component's constructor this entire PR would be free for compat - no bytes added lol. So close!